src/components/Calendar/Calendar.cy.tsx

```diff
@@ -23,55 +32,57 @@ describe("<Calendar />", () => {
             description: "a piece of description text",
         },
     ];
-
     it("calendar renders", () => {
-        cy.mount(<Calendar initialEvents={[]} />);
+        cy.mount(<StyledCalendar initialEvents={[]} />);
     });
 
     it("calendar is set to the current month when rendered in dayGridMonth", () => {
         const currentMonthYear = testDate.toLocaleDateString("en-GB", {
-            month: "long",
+            month: "short",
             year: "numeric",
         });
 
-        cy.mount(<Calendar initialEvents={[]} initialDate={testDate} />);
+        cy.mount(<StyledCalendar initialEvents={[]} initialDate={testDate} />);
         cy.get(".fc-toolbar-title").should("have.text", currentMonthYear);
     });
 
     it("events render", () => {
-        cy.mount(<Calendar initialEvents={sampleEvents} initialDate={testDate} />);
+        cy.mount(<StyledCalendar initialEvents={sampleEvents} initialDate={testDate} />);
         cy.get(".fc-event").should("be.visible");
     });
 
     it("can change view to timeGridDay", () => {
-        cy.mount(<Calendar initialEvents={[]} initialDate={testDate} />);
+        cy.mount(<StyledCalendar initialEvents={[]} initialDate={testDate} />);
         cy.get("button.fc-timeGridDay-button").click();
         cy.get(".fc-timeGridDay-view").should("be.visible");
     });
 
     it("can change view to timeGridWeek", () => {
-        cy.mount(<Calendar initialEvents={[]} initialDate={testDate} />);
+        cy.mount(<StyledCalendar initialEvents={[]} initialDate={testDate} />);
         cy.get("button.fc-timeGridWeek-button").click();
         cy.get(".fc-timeGridWeek-view").should("be.visible");
     });
 
     it("can change view between months in dayGridMonth", () => {
         const prevMonth = new Date(testDate);
         prevMonth.setMonth((testDate.getMonth() + 11) % 12);
-        const prevMonthYear = prevMonth.toLocaleString("en-GB", { month: "long", year: "numeric" });
+        const prevMonthYear = prevMonth.toLocaleString("en-GB", {
+            month: "short",
+            year: "numeric",
+        });
 
-        cy.mount(<Calendar initialEvents={[]} initialDate={testDate} />);
+        cy.mount(<StyledCalendar initialEvents={[]} initialDate={testDate} />);
         cy.get("button.fc-prev-button").click();
 
         cy.get(".fc-toolbar-title").should("have.text", prevMonthYear);
     });
```

Comment:
Also didn't know where to put it, but just thought I'd check you've seen the comments I added to the Jira ticket when technical reviewing previously?

Reply:
Yes I did